ecli/ecli-rs/build.rs

```diff
@@ -13,7 +13,7 @@ fn main() {
     println!("cargo:rustc-link-search=../../bpf-loader/build/libbpf");
 
     println!("cargo:rustc-link-lib=static=eunomia");
-    println!("cargo:rustc-link-lib=static=wasm-bpf");
+    println!("cargo:rustc-link-lib=static=wasm-bpf_LIB");
```

Comment:
In build.rs, I added the necessary link library `libwasm-bpf_LIB`, but it failed to link to the wasm_main function during the build process, the error is as follows:

```console
error: linking with `cc` failed: exit status: 1
   |
   = note: "cc" "-m64" "/tmp/rustc6Ga9bW/symbols.o" "/home/todo/bpf-worktree/fix-wrapper/ecli/ecli-rs/target/debug/deps/ecli-0b4f73099ea66c15. 10za9mji08vr70o1.rcgu.o"
.........
   = note: /usr/bin/ld: /home/todo/bpf-worktree/fix-wrapper/ecli/ecli-rs/target/debug/deps/ecli-0b4f73099ea66c15.52naf5190wgve4iw.rcgu.o: in function `ecli: :wasm_bpf_runner::wasm::handle_wasm':
           /home/todo/bpf-worktree/fix-wrapper/ecli/ecli-rs/src/wasm_bpf_runner/wasm.rs:47: undefined reference to `wasm_main'
           collect2: error: ld returned 1 exit status
          
   = note: some `extern` functions couldn't be found; some native libraries may need to be installed or have their path specified
   = note: use the `-l` flag to specify native libraries to link
   = note: use the `cargo:rustc-link-lib` directive to specify the native libraries to link with Cargo (see https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorustc- link-libkindname)

warning: `ecli` (bin "ecli") generated 17 warnings
error: could not compile `ecli` due to previous error; 17 warnings emitted
make: *** [Makefile:6: build] Error 101
```
It's confusing.